src/Bicep.Core.Samples/Files/Metadata_CRLF/main.sourcemap.bicep

```diff
@@ -17,6 +23,8 @@ metadata myMultiLineString = '''
 
 // object value
 metadata foo = {
+//@    "foo": {
+//@    },

```

Comment:
`foo` isn't empty in the compiled JSON. Are all container types being written out as empty? `myArrayMetadata` on line 71 below is also being included in this artifact as an empty array, but it's not empty in the JSON version.

Reply:
This sourcemap baseline format is admittedly a bit confusing - it's using the sourcemap to map each line of JSON output to a given line of Bicep (displayed **underneath** the line of Bicep).

Here we have - e.g.
* `"foo": {` is mapped back to `metadata foo = {`
* `"enabled": true,` is mapped back to `enabled: true`
* ...other props...
* `},` (the end of the metadata statement) is mapped back to `metadata foo = {`

Reply:
BTW the fact that both start and end of the object are mapped back to the same line I think is a limitation of the sourcemap format. We record data in terms of spans, so both are mapped back to the correct Bicep range. However the sourcemap file just stores line numbers - so this information is lost (we pick the first line number for the range).